src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -2370,4 +2446,25 @@ private class RD2_DataMigration_TEST {
         return Database.insert(new Opportunity(), false);
     }
 
+    /***
+    * @description Updates first two opportunities in the recurring donation to closed won for calculating rollups
+    */
+    private static void setRdOppsToClosedWonForRollupTests(npe03__Recurring_Donation__c rd, Integer numOpps) {
+        List<Opportunity> oppToUpdate = new List<Opportunity>();
+        List<Opportunity> oppList =  oppGateway.getRecordsByCloseDate(rd);
+
+        Integer count = 0;
+
+        for (Opportunity opp: oppList) {
+            if (count < numOpps) {
+                opp.StageName = UTIL_UnitTestData_TEST.getClosedWonStage();
+                opp.CloseDate = DATE_ESTABLISHED.addMonths(1);
```

Comment:
Why is the `CloseDate` reset for two oldest Opps? Date Established is 3 months ago from `today`, so there should not be any need to reset the `CloseDate`.

Reply:
I thought when you close an opportunity, it sets the close date as today, that's why changed to a past date.

Reply:
This happens only for future Opps not the past ones.

Reply:
When I checked debug logs, it is not resetting the closedates to today, so I will take it out then, good point,thanks!